release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -617,6 +617,88 @@ This action restores the prior functionality at the cost of globally reducing ne
+
+SELinux (Security-Enhanced Linux) is a security mechanism that assigns security labels (contexts) to all objects (files, processes, network ports, etc.) on a system. These labels determine what a process can access. When a pod starts, the container runtime recursively relabels all files on a volume to match a pod’s SELinux context. For large volumes, this can significantly increase pod startup times. 
+
+{product-title} 4.20 introduces the ability to set the parameter `selinuxChangePolicy` at both the namespace and pod level as a  generally available feature.
+
+//For more information about Configuring fsGroupChangePolicy and selinuxChangepolicy per namespace , see ???use same link as above RWX/RWO SELinux Mount???
+
+[id="ocp-release-notes-storage-rwx-rwo-selinux-mount_{context}"]
+==== RWX/RWO SELinux Mount (Technology Preview)
+Pods might take a significant amount of time to start when the volume contains a large number of files. To avoid SELinux labeling issues while keeping SELinux confining, you can enable the ReadWriteMany/ReadWriteOnce (RWX/RWO) SELinux Mount feature. With this feature, the persistent volume (PV) file system is mounted with the right SELinux context, thus avoiding relabeling every file. 
+
+This feature was introduced in {product-title} 4.17 as a Developer Preview feature, but is now supported in 4.20 as a Technology Preview feature.
+
+//For more information about RWX/RWO SELinux Mount, see ???
+
+[id="ocp-release-notes-storage-honor-vol-reclaim-policy_{context}"]
+==== Always honor persistent volume reclaim policy is generally available
+
+Prior to {product-title} 4.18, the persistent volume (PV) reclaim policy was not always applied.
+
+For a bound PV and persistent volume claim (PVC) pair, the ordering of PV-PVC deletion determined whether the PV delete reclaim policy was applied or not. The PV applied the reclaim policy if the PVC was deleted prior to deleting the PV. However, if the PV was deleted prior to deleting the PVC, then the reclaim policy was not applied. As a result of that behavior, the associated storage asset in the external infrastructure was not removed.
+
+Starting with {product-title} 4.18, the PV reclaim policy is consistently always applied as a Technical Preview feature. With {product-title} 4.20, this feature is generally available.
+
+For more information, see xref:../storage/understanding-persistent-storage.adoc#reclaiming_understanding-persistent-storage[Reclaim policy for persistent volumes].
+
+[id="ocp-release-notes-storage-csi-manila-multiple-cdir_{context}"]
+==== Manila CSI driver allows multiple CIDRs when creating NFS volumes is generally available
+By default, {product-title} creates Manila storage classes that provide access to all IPv4 clients. In {product-title} 4.20, you can limit client access by defining custom storage classes that use specific client IP addresses or subnets by using the `nfs-ShareClient` parameter.
```

Comment:
So it's not completely correct, it was possible to limit client access but you could only set one CIDR, this new feature allows to configure multiple entries.
@gouthampacha

Reply:
works for me thanks!

Reply:
@gcharot @silvacarloss Fixed. But I assume that I need to update the docs content as well.

Reply:
Nope, I don't see anything wrong in the content added with https://github.com/openshift/openshift-docs/pull/99555/

Reply:
yeh same here, the doc PR does not mention what was possible in the previous release so all good

Reply:
OK. Then I'm leaving it as is.